response_operations_ui/views/reporting_units.py

```diff
@@ -188,21 +183,21 @@ def view_reporting_unit_survey(ru_ref, survey_id):
 
     attributes = party_controller.get_business_attributes_by_party_id(reporting_unit["id"])
     collection_exercises_with_details = [
-        add_collection_exercise_details(ce, attributes[ce["id"]], case_groups) for ce in survey_collection_exercises
+        add_collection_exercise_details(ce, attributes[ce["id"]], ru_details) for ce in survey_collection_exercises
     ]
 
     survey_details = get_survey_by_id(survey_id)
     survey_details["display_name"] = f"{survey_details['surveyRef']} {survey_details['shortName']}"
 
     # If there's an active IAC on the newest case, return it to be displayed
     live_collection_exercise_ids = [ce["id"] for ce in survey_collection_exercises]
-    valid_cases = [
-        case for case in cases if case.get("caseGroup", {}).get("collectionExerciseId") in live_collection_exercise_ids
-    ]
-    case = next(iter(sorted(valid_cases, key=lambda c: c["createdDateTime"], reverse=True)), None)
+    live_rus = [ru for ru in ru_details if ru.get("collectionExerciseId") in live_collection_exercise_ids]
```

Comment:
As the list of objects being iterated are casegroup/cases, the name `live_rus` is misleading.